pkg/kubelet/cm/devicemanager/manager_test.go

```diff
@@ -282,8 +282,8 @@ func setup(t *testing.T, devs []*pluginapi.Device, callback monitorCallback, soc
 
 func setupInProbeMode(t *testing.T, devs []*pluginapi.Device, callback monitorCallback, socketName string, pluginSocketName string) (Manager, <-chan interface{}, *Stub, pluginmanager.PluginManager) {
 	m, updateChan := setupDeviceManager(t, devs, callback, socketName)
-	pm := setupPluginManager(t, pluginSocketName, m)
 	p := setupDevicePlugin(t, devs, pluginSocketName)
+	pm := setupPluginManager(t, pluginSocketName, m)
```

Comment:
This change seems OK -- just curious if you know exactly why the order matters here.

Reply:
See the explain before.  Previous order may cause  device-plugin.sock add twice.  

one is from 
https://github.com/kubernetes/kubernetes/blob/970b532379b2b1d0b1352f960fd7f88f0726123e/pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go#L71-L80

another is from
https://github.com/kubernetes/kubernetes/blob/970b532379b2b1d0b1352f960fd7f88f0726123e/pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go#L119-L145

Reply:
Makes sense. Thanks for clarifying